### DISCUSSION TITLE
Is there anyway sync data between remote sqlite and local sqlite

### DISCUSSION THREAD

**Discussion by bilalswiftsolutions** [NativePHP/laravel#319] [Q&A] [answered]
### What were you trying to do?

want to share data between all users using applicatiom

### What happened?

unable to shared data between 2 users

### How to reproduce the bug

"Not A BUG"

### Package Versions

"version": "0.6.3",

### PHP Version

8.1.25

### Laravel Version

10

### Node Version

20.11.1

### Which operating systems have you seen this occur on?

Windows

### OS version

23H2

### Notes

 i have multiple user using application as desktop app, data need to be share between all users, as sqlite only database we can use with native php , is there anyway we can replicate db between dbs

**Comment by simonhamp** [comment, marked as answer]
I've moved this to the Q&A discussion board, as it feels more appropriate here.

SQLite is just the default driver. You could [build the PHP binaries yourself](https://static-php.dev/) with support for the relevant database you want to use (if it's supported for static builds).

Please note though that shipping custom binaries isn't supported and something we're discouraging because of the risk it presents to end users.

For now, there are no plans to add other database drivers to the default NativePHP binaries, so your best option is to set up a centralised service - e.g. a Rest API - that your app can talk to.
